src/pages/About.jsx

```diff
@@ -70,24 +72,38 @@ const team = [
     ),
   },
 ];
+
+const aboutText = `DreamScholar looks to make it easier for people to pay for school. Our mission is to remove the barriers that prevent and discourage students from pursuing higher education due to difficulties finding scholarships that they qualify for. Meet the team behind DreammScholar. Want to get involved? Shoot us an email!
+
+Meet the team behind DreamScholars. We are a group of lifelong learners who are passionate about technology and community upliftment. The project arose from our own struggles as students and the lack of resources for our community.  
+`;
+
+const useStyles = makeStyles((theme) => ({
+  aboutText: {
+    marginBottom: theme.spacing(2),
+    whiteSpace: 'pre-line',
```

Comment:
FYI instead of this you could also have two **Typography** paragraph elements with the two paragraphs hard-coded inside them. For adding the margin below it looks like setting either the `gutterBottom` or `paragraph` props can work for you according to the [API](https://material-ui.com/api/typography/#props)